scripts/install.sh

```diff
@@ -62,14 +62,7 @@ cat $SCRIPT_DIR/running.status.json | sed s/@@DATE@@/$(date -u +%FT%TZ)/ > $stat
 echo "Installing Docker..."
 
 if [ $distrib_id == "Ubuntu" ]; then
-    export DEBIAN_FRONTEND=noninteractive
-    apt-get update
-    apt-get install -y -q linux-image-extra-$(uname -r) apt-transport-https
-    echo deb https://get.docker.com/ubuntu docker main > /etc/apt/sources.list.d/docker.list
-    apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 36A1D7869245C8950F966E92D8576A8BA88D21E9
-    sleep 3
-    apt-get update
-    apt-get install -y -q lxc-docker
+    curl -sSL https://get.docker.com/ | sh
```

Comment:
https://docs.docker.com/installation/ubuntulinux/ this guide advises:

    $ wget -qO- https://get.docker.com/ | sh

for me `curl -sSL` is fine too but let's stick with the guide.